### DISCUSSION TITLE
tab activity symbol color

### DISCUSSION THREAD

**Discussion by GG2R10** [kovidgoyal/kitty#6243] [Q&A]
Hey, I really want to change the color of tab_activity_symbol but I can't find the option to change it, it's actually red and I'd like to make it white. Is there a simple way to do it, could you make an example of what I have to do? Thanks.

![image](https://user-images.githubusercontent.com/114362342/236365537-9263447c-a3f0-439a-8475-9df6ec82e1fd.png)

**Comment by kovidgoyal** [comment]
Read the documentation of tab_title_template

**Reply by GG2R10** [reply]
wow, I hadn't read that part correctly, thanks!
![image](https://user-images.githubusercontent.com/114362342/236380924-9d1b16f5-b33c-47ef-b9ac-f63ac4dfd170.png)
